templates/launch.html

```diff
@@ -39,28 +37,13 @@ <h1>Launch a survey</h1>
             </optgroup>
         </select>
     </div>
-    <div class="field-container">
-        <label for="exp">Token Expiry (seconds)</label>
-        <input id="exp" name="exp" type="text" value="1800" class="qa-token-expiry">
-    </div>
-
-    <div class="field-container">
-        <label for="period_str">Period String</label>
-        <input id="period_str" name="period_str" type="text" value="May 2016" class="qa-period-str">
-    </div>
 
-    <div class="field-container">
-        <label for="period_id">Period ID</label>
-        <input id="period_id" name="period_id" type="text" value="201605" class="qa-period-id">
+    <h3>Survey Metadata</h3>
+    <div id="survey_metadata">
+        <p>--- Select Schema ---</p>
```

Comment:
Could this either:
a) say something more clear like "Metadata fields will be loaded when you select a schema". Or,
b) Not show the heading until the metadata is selected?

It's just not guiding the user through it as well as it could at the moment. Should save everyone asking you what to do when it's released.

Reply:
👍